From: Jiri Bohac <jbohac@suse.cz>
Patch-mainline: no
Subject: enable nf_conntrack accounting by default

Keep the SLE-11 SP1 default value, formerly done by the
CONFIG_NF_CONNTRACK_ACCT config option.

diff --git a/Documentation/kernel-parameters.txt b/Documentation/kernel-parameters.txt
index 301c028..3a4a1cb 100644
--- a/Documentation/kernel-parameters.txt
+++ b/Documentation/kernel-parameters.txt
@@ -1579,7 +1579,7 @@ bytes respectively. Such letter suffixes can also be entirely omitted.
 			[NETFILTER] Enable connection tracking flow accounting
 			0 to disable accounting
 			1 to enable accounting
-			Default value is 0.
+			Default value is 1.
 
 	nfsaddrs=	[NFS] Deprecated.  Use ip= instead.
 			See Documentation/filesystems/nfs/nfsroot.txt.
diff --git a/net/netfilter/nf_conntrack_acct.c b/net/netfilter/nf_conntrack_acct.c
index 5178c69..43e4cb3 100644
--- a/net/netfilter/nf_conntrack_acct.c
+++ b/net/netfilter/nf_conntrack_acct.c
@@ -17,7 +17,7 @@
 #include <net/netfilter/nf_conntrack_extend.h>
 #include <net/netfilter/nf_conntrack_acct.h>
 
-static int nf_ct_acct __read_mostly;
+static int nf_ct_acct __read_mostly = 1;
 
 module_param_named(acct, nf_ct_acct, bool, 0644);
 MODULE_PARM_DESC(acct, "Enable connection tracking flow accounting.");
